### PR TITLE
update manager and collector logback config(#1704)

### PR DESCRIPTION
**Pull request by handy-git**
## Checklist

- [x]  I hereby agree to the terms of the [HertzBeat CLA](https://gist.github.com/tomsun28/511c04e7643901cb550bb6ecc75a661b)
- [x]  I have read the [Contributing Guide](https://hertzbeat.com/docs/others/contributing/)
- [ ]  I have written the necessary doc or comment.
- [ ]  I have added the necessary unit tests and all cases have passed.

## Add or update API

- [ ] I have added the necessary [e2e tests](../e2e) and all cases have passed.

### CODE REVIEW

File: manager/src/main/resources/logback-spring.xml [hunk 1]
```diff
@@ -94,7 +94,7 @@
     <!-- 生产环境配置 -->
     <springProfile name="prod">
         <root level="INFO">
-            <appender-ref ref="ConsoleAppender"/>
+            <!--<appender-ref ref="ConsoleAppender"/>-->
```
Comment (tomsun28) [inline manager/src/main/resources/logback-spring.xml:97]:
```suggestion
```

**Review by tomsun28 — COMMENTED** [review]
👍 suggest remove ConsoleAppender in collector `<springProfile name="cluster">`

**Comment by handy-git** [comment]
> 👍 suggest remove ConsoleAppender in collector `<springProfile name="cluster">`

I've taken care of it.

**Review by tomsun28 — APPROVED** [review]

**Comment by tomsun28** [comment]
👍👍 hi, one more thing. Can you help us update the chinese desc to english desc in these file.  eg: ```<!--日志保留时长-->```

**Comment by handy-git** [comment]
> 👍 suggest remove ConsoleAppender in collector `<springProfile name="cluster">`

ok

**Pull request merged**